lib/msal-common/test/cache/MockCache.ts

```diff
@@ -124,6 +124,23 @@ export class MockCache {
         const atThree = CacheManager.toObject(new AccessTokenEntity(), atThreeData);
         this.cacheManager.setAccessTokenCredential(atThree);
 
+        // BEARER with AuthScheme Token
```

Comment:
Is this name right? I'd assume bearer tokens still have no auth scheme in the cache key to prevent backwards incompatible changes?

Reply:
hmm I thought everything should be with auth scheme explicitly now even if we still support and know how to read old entries without. Is that not the case?

In any case, nothing changed functionally around this in the library, this code here is just testing all combinations and scenarios.

Reply:
No, bearer tokens still use the regular -accesstoken- key signature. We could make that change for v3 since it's a breaking change.

Reply:
I guess "should" is the right word to use here, it's not an extensive change, I'll create a ticket.